podpac/core/settings.py

```diff
@@ -19,8 +19,9 @@
     'DEBUG': False,  # This flag currently sets self._output on nodes
     'DEFAULT_CACHE': ['ram'],
     'CACHE_OUTPUT_DEFAULT': True,
-#    'RAM_CACHE_MAX_BYTES': 1e9, # ~1GB    TODO
-#    'DISK_CACHE_MAX_BYTES': 10e9, # ~10GB TODO
+    'RAM_CACHE_MAX_BYTES': 1e9, # ~1GB    TODO
+    'DISK_CACHE_MAX_BYTES': 10e9, # ~10GB TODO
+    'S3_CACHE_MAX_BYTES': 10e9, # ~10GB TODO
```

Comment:
Let's make this 100 GB, and remove the TODO's on the comments.

Reply:
Actually, 10GB is fine as a default (more conservative). Just need to remove the TODOS.

Reply:
Yeah, thanks. One thing about how this is working, the user would have to explicitly set these limits in the settings to `None` in order to achieve unlimited storage. If they leave the setting out it will go to the defaults. There is a performance hit to using any limit (especially with S3).

I should put a note about `None`. Would this go in the `settings.py` comments?

Reply:
Added not about `None` to `PodpacSettings` class.